src/test/java/com/plaid/client/integration/PaymentCreateTest.java

```diff
@@ -19,7 +19,7 @@
    */
   public static Response<PaymentCreateResponse> createPayment(PlaidClient client) throws Exception {
 
-    Response<RecipientCreateResponse> createRecipientResponse = RecipientCreateTest.createRecipient(client);
+    Response<RecipientCreateResponse> createRecipientResponse = RecipientCreateTest.createRecipientWithIban(client);
```

Comment:
Debated also adding a test for the "createRecipientWithBacs" test case but reasoned that because we have tests that assert that recipient creation is successful, the payment creation tests should only cover parts of the flow post recipient creation

Reply:
Makes sense, is there a `createRecipientWithBacs` function somewhere that I missed?

Reply:
https://github.com/plaid/plaid-java/pull/228/files#diff-e05ab5e9434e0f7c11bbb7b29974119bR38